pkg/client/k8s/k8s.go

```diff
@@ -570,7 +571,8 @@ func (k *K8sClient) DeleteAppsV1B1Deployment(name string) error {
 
 func getK8sConfig() (config *rest.Config, err error) {
 	k8sMaster := api_oe_old.K8sMasterENV()
-	kubeConfig := api_oe_old.KubeConfigENV()
+	// kubeConfig := api_oe_old.KubeConfigENV()
+	kubeConfig := os.Getenv("KUBECONFIG")
 
```

Comment:
can you please mention why you commented the above code and why have you used os.Getenv()? I believe there are certain scenarios for which the commented code stands valid.

Reply:
it is because on running mayactl outside the pod we will be needing  kubeconfig to make request from kube api-server

Reply:
Handle this part in case of using external kubeconfig from outside the maya-apiserver pod.